docs/customize/vocabularies/subjects.md

```diff
@@ -0,0 +1,21 @@
+# Subjects (MeSH)
```

Comment:
@fenekku can you crosscheck Im not saying anything completly wrong in this section 👼

Reply:
I will just explain how to add a custom subjects here and point to the MeSH one as an example.